### ISSUE TITLE
[Feature] Allow Multiple Users,...

### ISSUE THREAD

**Issue by weltmeyer** [hooram/ownphotos#29]
I would really like to use OwnPhotos for more than one user.

We would need different user-logins and datafolders.
Best would be if we put this in the database somewhere.

This way, we could also easily integrate it with nextcloud, if we can use nextcloud as a login provider. 

#19

**Comment by petrk94** [comment]
+1

**Comment by hooram** [comment]
https://github.com/hooram/ownphotos/issues/27

**Comment by MarPiRK** [comment]
It would be nice to design authentication system with various authentication methods in mind. I'm mostly interested in Nextcloud integration, so it would be nice to have OpenID Connect support, but other auth methods will probably be welcomed by others in future (like SAML, LDAP).

**Comment by adocampo** [comment]
Yes, multi-user would be critical to publish this on the Internet. I up vote for an LDAP connector, besides the local DB users.

**Comment by guysoft** [comment]
There is an option to open signup at the moment.

**Comment by pwFoo** [comment]
Would be nice to use the nextcloud users to login and manage images and galleries?
